Source/WebKit/Platform/IPC/unix/ConnectionUnix.cpp

```diff
@@ -43,26 +43,17 @@
 #include <wtf/TZoneMallocInlines.h>
 #include <wtf/UniStdExtras.h>
 
-#if USE(GLIB)
-#include <gio/gio.h>
-#include <wtf/glib/GUniquePtr.h>
-#endif
-
 #if OS(DARWIN)
 #define MSG_NOSIGNAL 0
 #endif
 
 // Although it's available on Darwin, SOCK_SEQPACKET seems to work differently
-// than in traditional Unix so fallback to STREAM on that platform.
+// than in traditional Unix so fallback to DGRAM on that platform.
 #if defined(SOCK_SEQPACKET) && !OS(DARWIN)
 #define SOCKET_TYPE SOCK_SEQPACKET
 #else
-#if USE(GLIB)
-#define SOCKET_TYPE SOCK_STREAM
-#else
 #define SOCKET_TYPE SOCK_DGRAM
 #endif
-#endif // SOCK_SEQPACKET
```

Comment:
I think you can delete all this too, for the same reason.

`SOCKET_TYPE` is notably only used in one place, and there it should always be `SOCK_DGRAM` because the GLib and Darwin conditions probably won't ever be true after this. All other calls to `createPlatformConnection` use hardcoded `SOCK_SEQPACKET`, so you could also remove the socket type parameter of `createPlatformConnection` entirely if you want to.

Reply:
(Feel free to ignore this if you don't want to change it.)